docs/Makefile

```
@@ -1,8 +1,9 @@
 lint:
-	go run -C ../misc/docs/tools/linter . -path "$(PWD)" -treat-urls-as-err=true
+	go run -C ../misc/docs/tools/linter . -path $(PWD) -treat-urls-as-err=true
 
 generate:
 	go run -modfile ../misc/devdeps/go.mod github.com/campoy/embedmd -w `find . -name "*.md"`
+	go run -C ../misc/docs/tools/indexparser . -path $(PWD)/README.md > ./_assets/sidebar.json
```

Can you move this file tob’
```suggestion
	go run -C ../misc/docs/tools/indexparser . -path $(PWD)/README.md > ../misc/docs/sidebar.json
```

---

[612af48](https://github.com/gnolang/gno/pull/4004/commits/612af4817d013d44323640e1ae6eec764a823662)